File: Python/Problem-17.py
# Write a Program to print the Armstrong Number

number = int(input("Enter Any Number: "))
temp = number
number_length = len(str(temp))
s = 0
while temp > 0:
  remender = temp % 10
  s += remender ** number_length
  temp = temp // 10

if s == number:
  print(f"{number} is a Amstrong Number")
else:
  print(f"{number} is not a Armstrong Number")

File: JavaScript/Problem-17.js
// # Write a Program to print the Armstrong Number
const prompt = require("prompt-sync")();
const number = prompt("Enter Any Number: ");
var temp = number;
const number_length = temp.length;
console.log(number_length)
var s = 0;
while (temp > 0) {
  let remender = temp % 10;
  s += remender ** number_length;
  temp = parseInt(temp / 10)
}

if (s == number) {
  console.log(`${number} is a Amstrong Number`);
} else {
  console.log(`${number} is not a Armstrong Number`);
}
